core/test/financial_templates/TokenFactory.js

```diff
@@ -35,8 +37,30 @@ contract("TokenFactory", function(accounts) {
       from: tokenCreator
     });
     const token = await Token.at(tokenAddress);
-    assert(await token.isMinter(tokenCreator));
+
+    // Creator should be only minter
     assert(!(await token.isMinter(contractDeployer)));
+    assert(await token.isMinter(tokenCreator));
+
+    // Creator should be only burner
+    assert(!(await token.isBurner(contractDeployer)));
```

Comment:
nit: prefer `assert.isFalse` instead of `assert(!...`

Reply:
done